### PR TITLE
Update install-ocaml script for ocaml 4.05.0

### PR DESCRIPTION
**Pull request by binji**

### CODE REVIEW

File: interpreter/meta/travis/install-ocaml.sh [hunk 1]
```diff
@@ -12,16 +12,29 @@ cd $(dirname ${BASH_SOURCE[0]})/../../..
 rm -rf ocaml
 mkdir ocaml
 cd ocaml
-curl https://wasm.storage.googleapis.com/ocaml-4.02.2.tar.gz -O
-CHECKSUM=$(shasum -a 256 ocaml-4.02.2.tar.gz | awk '{ print $1 }')
+curl https://wasm.storage.googleapis.com/ocaml-4.05.0.tar.gz -O
```
Comment (dschuff) [inline interpreter/meta/travis/install-ocaml.sh]: It seems odd to have one tarball in google storage and the other from github. We might as well just put them both in google storage, or switch this to https://github.com/ocaml/ocaml/archive/4.05.0.tar.gz ?

File: interpreter/meta/travis/install-ocaml.sh [hunk 2]
```diff
@@ -12,16 +12,29 @@ cd $(dirname ${BASH_SOURCE[0]})/../../..
...
+CHECKSUM=$(shasum -a 256 0.11.0.tar.gz | awk '{ print $1 }')
+if [ ${CHECKSUM} != \
+  1717edc841c9b98072e410f1b0bc8b84444b4b35ed3b4949ce2bec17c60103ee ]; then
+  echo "Bad checksum ocaml download checksum!"
+  exit 1
+fi
+tar xfz 0.11.0.tar.gz
+cd ocamlbuild-0.11.0
+make configure
+make install
```
Comment (dschuff) [inline interpreter/meta/travis/install-ocaml.sh]: Does this work? Don't you need to set the prefix here too, and add the ocaml install dir to the path?

Comment (binji) [reply interpreter/meta/travis/install-ocaml.sh]: Yeah, you're right.

**Comment by binji** [comment]
Yeah, I was just doing that actually :-)

File: interpreter/meta/travis/install-ocaml.sh [hunk 3]
```diff
@@ -2,26 +2,45 @@
...
+cd ..
+
+PATH=${PREFIX}/bin:${PATH}
+
+download_from_gh_archive ocamlbuild 0.11.0 1717edc841c9b98072e410f1b0bc8b84444b4b35ed3b4949ce2bec17c60103ee
+cd ocamlbuild-0.11.0
+make configure
+make
 make install
```
Comment (dschuff) [inline interpreter/meta/travis/install-ocaml.sh]: Do you need `OCAMLBUILD_PREFIX=${PREFIX}` here or does it pick that up from ocaml itself?

Comment (binji) [reply interpreter/meta/travis/install-ocaml.sh]: It pulls it from ocaml. In fact, I tried setting it manually but it didn't seem to work properly.

Comment (dschuff) [reply interpreter/meta/travis/install-ocaml.sh]: Odd, that's what I did in https://github.com/WebAssembly/waterfall/pull/244

**Comment by dschuff** [comment]
anyway, this still LGTM

File: interpreter/meta/travis/install-ocaml.sh (hunk 3 above)
Comment (binji) [reply interpreter/meta/travis/install-ocaml.sh]: Hm, I probably screwed something up. :-) Oh well.

**Pull request merged**